### DISCUSSION TITLE
HELP! Color ICC Profies for ROG Strix SCAR 16 2024 Needed!

### DISCUSSION THREAD

**Discussion by Tozward** [seerge/g-helper#2845] [Q&A] [answered]
I'm swapping to NE160QDM-NM7 panel used on SCAR 16 2024, but on a non-ASUS laptop (which disabled me to utilize Armoury Crate to download the color calibration profiles). After searching through the Internet, I found some Reddit posts mentioning them but the download links they provided had expired.

Though @n1gq has provided screenshots of the files I'm looking for in [another discussion](https://github.com/seerge/g-helper/discussions/2190), I'm not sure whether he could notice my reply.
![image](https://github.com/user-attachments/assets/681ad6db-7bf5-477f-8911-6ba58868da4b)

Could anyone share the ICC Profies for ROG Strix SCAR 16 2024 (G634JZR/G634JYR) with me? Replies will be highly appreciated!

**Comment by IceStormNG** [comment]
This is for the G634JZR: https://dlcdn-rogboxbu1.asus.com/pub/ASUS/APService/Gaming/SYS/ROGS/19109-R5WFFH-76cdb2bad9582d23c1f6f4d868218d6c.zip

And this is for G634JYR: https://dlcdn-rogboxbu1.asus.com/pub/ASUS/APService/Gaming/SYS/ROGS/19110-FPFKO4-76cdb2bad9582d23c1f6f4d868218d6c.zip

**Reply by Tozward** [reply]
Thanks for the relpy! However, both the zip files are EMPTY!! I'm using WinRAR

**Reply by IceStormNG** [reply]
You're right... I will check the other versions later. Maybe they do not deliver these profiles anymore through this API, after we found out how it works 😄

**Reply by IceStormNG** [reply, marked as answer]
Alright, try this one. This has color profiles included:
https://dlcdn-rogboxbu1.asus.com/pub/ASUS/APService/Gaming/SYS/ROGS/19130-MUAT65-0eced7ca55a61cc41341faa3a48d34f3.zip

**Reply by Tozward** [reply]
Many thanks! @IceStormNG

**Reply by zerosmasher** [reply]
> Alright, try this one. This has color profiles included: https://dlcdn-rogboxbu1.asus.com/pub/ASUS/APService/Gaming/SYS/ROGS/19130-MUAT65-0eced7ca55a61cc41341faa3a48d34f3.zip

Do you have the color profile for the ROG Strix G16 G614JV-N4369W @IceStormNG

**Reply by IceStormNG** [reply]
> Do you have the color profile for the ROG Strix G16 G614JV-N4369W @IceStormNG

https://dlcdn-rogboxbu1.asus.com/pub/ASUS/APService/Gaming/SYS/ROGS/19128-H97T5E-c8c4756456d76c84a7ed6c3d8d050c8b.zip

Should be this one

**Reply by zerosmasher** [reply]
> > Do you have the color profile for the ROG Strix G16 G614JV-N4369W @IceStormNG
> 
> https://dlcdn-rogboxbu1.asus.com/pub/ASUS/APService/Gaming/SYS/ROGS/19128-H97T5E-c8c4756456d76c84a7ed6c3d8d050c8b.zip
> 
> Should be this one

Wow, many thx bro. Btw, I installed a new screen for my rog laptop, not from the asus center. The screen clearly show that it has 240hz but when I check it on my setting, it sometimes show the 240hz option, sometimes doesn't. What could be the problem with that.
Many thx @IceStormNG

**Reply by MinbinLuo** [reply]
Hi @IceStormNG, do you have the color profile for `ROG Strix SCAR 16 (2025)G635LW-XS97` ?
Many thanks!

**Reply by IceStormNG** [reply]
> Hi @IceStormNG, do you have the color profile for `ROG Strix SCAR 16 (2025)G635LW-XS97` ? Many thanks!

https://dlcdn-rogboxbu1.asus.com/pub/ASUS/APService/Gaming/SYS/ROGS/21985-AV5G50-9b3da7234e21be309ec3d889cd88c4af.zip

This should be the one.

**Reply by kirayamatosf** [reply]
Do you have the color profile for the ROG zephyrus G16 2025 GU605CX? Much thanks.

**Reply by IceStormNG** [reply]
> Do you have the color profile for the ROG zephyrus G16 2025 GU605CX? Much thanks.

Here you go: https://dlcdn-rogboxbu1.asus.com/pub/ASUS/APService/Gaming/SYS/ROGS/22122-M8J9C2-029d307fdbc49dd80dca79e1c8ed6e32.zip

**Reply by pawelm8** [reply]
Do you happen to have the color profiles for Scar 18 2025 G835LW-SA032W?
Thank you in advance.

**Reply by IceStormNG** [reply]
> Do you happen to have the color profiles for Scar 18 2025 G835LW-SA032W? Thank you in advance.

https://dlcdn-rogboxbu1.asus.com/pub/ASUS/APService/Gaming/SYS/ROGS/21989-JGI499-89ec59aa686a58d890e6f441c329e8bd.zip

**Reply by pawelm8** [reply]
> > Do you happen to have the color profiles for Scar 18 2025 G835LW-SA032W? Thank you in advance.
> 
> https://dlcdn-rogboxbu1.asus.com/pub/ASUS/APService/Gaming/SYS/ROGS/21989-JGI499-89ec59aa686a58d890e6f441c329e8bd.zip

Perfect! Thank you, much appreciated.

**Reply by evenstar40** [reply]
Hi @IceStormNG really appreciate all you do to help with these color profiles. Do you happen to have one for this model? Rog Strix G16 (2025) G635LX-XS97

**Reply by seerge** [reply]
@evenstar40 you can just download [latest release of the app](https://github.com/seerge/g-helper/releases), it can download any color profile for you even for 2025 models.

**Reply by Snapze** [reply]
> Alright, try this one. This has color profiles included: https://dlcdn-rogboxbu1.asus.com/pub/ASUS/APService/Gaming/SYS/ROGS/19130-MUAT65-0eced7ca55a61cc41341faa3a48d34f3.zip

Is it possible for you to share me the g634jzr color profiles and pqconfig?

**Reply by IceStormNG** [reply]
> g634jzr

https://dlcdn-rogboxbu1.asus.com/pub/ASUS/APService/Gaming/SYS/ROGS/19131-U3MKE0-8df343ff4f489beb7b267822116a4e1b.zip

The PQ Config is not in there. I'm not 100% sure how to get it though. You might need the Dolby Vision Decoder package from the Microsoft Store to generate that file.
Did you install the displayHDR Driver from ASUS for your machine? Maybe that one contains the file.

**Reply by Snapze** [reply]
I have the g634jz pqconfig, but I upgraded the display that g634jyr and g634jzr have and im trying to make dolby vision and then gsync in dgpu mode. If I leave the color profile from my previous OG display, dolby works but I rather use color profiles for the new display. Any tips? How can make this work? I installed the DisplayHdr driver from g634jzr and the smart display control from that laptop. Today ill try to install the intel gpu driver aswell from the newer laptop.

**Reply by emowpee** [reply]
Hi @IceStormNG can you provide link for Asus Scar 16 (2025) G635 ? 

I am planning on upgrading to NE160QDM-NM8 soon and seems this is the laptop match based on my _little_ researching. Thanks in advance!

**Reply by emowpee** [reply]
> @evenstar40 you can just download [latest release of the app](https://github.com/seerge/g-helper/releases), it can download any color profile for you even for 2025 models.

Hi @seerge, how can I do this? For context, I currently have GU603VV and has been meaning to download G635xx profiles for NE160QDM-NM8 panel.

**Reply by IceStormNG** [reply]
There aren't any color profiles for the Model G635. The ZIP files are empty.

**Comment by Jack32X** [comment]
Sorry to necro this, but I have a Strix 16 2023 (G614JZ) that I've upgraded to the boe ne160qdm nm7. Which colour profiles do I need (fresh windows install) and how do I install them, I got game visual menu to "work" but it doesn't actually change anything no matter what I select. Many thanks!

**Reply by seerge** [reply]
@Jack32X since all visual mode / gamuts are handled by AsusSplendid utility under the hood, likely it simply won't work at all. It assumes standard display panel that is installed from the factory. If you have your own - asus tool will ignore that.

**Comment by iamveryverystupid** [comment]
i have a 2023 strix scar 16, g634jy, can someone give me icc profiles for NE160QDM-NZ5?  im on linux so i cant get them from armoury crate.  thanks
